### PR TITLE
AND-6862 clean share sample

### PR DESCRIPTION
**Pull request by nyjalusc**
Addressed all the comments from https://github.com/box/box-android-share-sdk/pull/66

Created a new branch to keep the commit history clean. There was some junk in the old branch.

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @nyjalusc has signed the CLA. Thanks for the pull request!

**Comment by rgcorrea** [comment]
I believe you can remove the Roboto-Medium.ttf binary file.

File: box-share-sdk/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <resources>
     <string name="box_sharesdk_name">Share</string>
+    <string name="box_sharesdk_sample_name">Share SDK Sample</string>
+    <string name="box_sharesdk_sample_choose_action" description="description of available actions for a folder">Choose action for \"%s\"</string>
```
Comment (rgcorrea) [inline box-share-sdk/src/main/res/values/strings.xml]: Does the expected string have the double quotes on the UI ?

File: box-share-sdk/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,5 +1,7 @@
 <resources>
     <string name="box_sharesdk_name">Share</string>
+    <string name="box_sharesdk_sample_name">Share SDK Sample</string>
```
Comment (seema-at-box) [inline box-share-sdk/src/main/res/values/strings.xml]: Please add descriptions for the strings..so, our localization team knows what they mean.

File: box-share-sdk/src/main/res/values/strings.xml (hunk 1 above)
Comment (nyjalusc) [inline box-share-sdk/src/main/res/values/strings.xml]: Yes it does. I am doing it to make the name of the folder explicitly clear.

File: box-share-sdk/src/main/res/values/strings.xml (hunk 2 above)
Comment (nyjalusc) [inline box-share-sdk/src/main/res/values/strings.xml]: +1

**Comment by seema-at-box** [comment]
+1

**Pull request merged**